### PR TITLE
[BUGFIX] Make route-enhancer configuration work

### PR DESCRIPTION
**Pull request by benjaminkott**
Releases: master, 9.1, 9.0

### CODE REVIEW

**Comment by benjaminkott** [comment]
Needs to be adjusted after https://github.com/TYPO3GmbH/blog/issues/31 is done.

File: Configuration/Routes/BlogArchive.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              maerz: 3
+              april: 4
+              mai: 5
+              juni: 6
+              juli: 7
+              august: 8
+              september: 9
+              oktober: 10
+              november: 11
+              dezember: 12
```
Comment (romm) [inline Configuration/Routes/BlogArchive.yaml]: French translation for months below. 🙂 

```suggestion
              dezember: 12
            locale: 'fr_.*'
            map:
              janvier: 1
              fevrier: 2
              mars: 3
              avril: 4
              mai: 5
              juin: 6
              juillet: 7
              aout: 8
              septembre: 9
              octobre: 10
              novembre: 11
              decembre: 12
```

**Comment by benjaminkott** [comment]
thx @romm

File: Configuration/Routes/BlogArchive.yaml [hunk 2]
```diff
@@ -55,6 +55,7 @@ routeEnhancers:
               oktober: 10
               november: 11
               dezember: 12
+          -
```
Comment (romm) [inline Configuration/Routes/BlogArchive.yaml]: Oops!

Comment (benjaminkott) [reply Configuration/Routes/BlogArchive.yaml]: 😘

**Pull request merged**